tests/providers/google/cloud/hooks/test_pubsub.py

```diff
@@ -58,6 +60,21 @@ def setUp(self):
                         new=mock_init):
             self.pubsub_hook = PubSubHook(gcp_conn_id='test')
 
+    def _generate_messages(self, count):
+        return [
+            ParseDict(
+                {
+                    "ack_id": "%s" % i,
```

Comment:
```suggestion
                    "ack_id": str(i),
```
Does it have to be a string?